### PR TITLE
Created Author classes and made slight changes to the Book classes.

### PR DESCRIPTION
**Pull request by d-herrero**

### CODE REVIEW

**Review by pabloehc — APPROVED** [review]
Lgtm

File: src/main/java/bookstore/model/Author.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    @Column
+    private String firstName;
+    @Column
+    private String lastName;
+
+    @Column
+    private String publisher;
+    @Column
+    private String biography;
+
```
Comment (Mherr162) [inline src/main/java/bookstore/model/Author.java:20]: Add a default constructor and another constructor with all fields but id

File: src/main/java/bookstore/service/BookService.java [hunk 2]
```diff
@@ -11,6 +12,7 @@ public interface BookService {
     public String deleteBook(String bookId);
     public Optional<Book> getBook(Long bookId);
     public List<Book> getAllCBooks();
-    public List<Book> getByISBN(String isbn);
+    public List<Book> findByISBN(String isbn);
+    public List<Book> findByAuthor(String author);
```
Comment (Mherr162) [inline src/main/java/bookstore/service/BookService.java:16]: For List<Book> findByAuthor, replace String with Long

File: src/main/java/bookstore/service/impl/AuthorService.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package bookstore.service.impl;
+
+import bookstore.model.Author;
+import bookstore.repository.AuthorRepository;
+
+public class AuthorService
```
Comment (Mherr162) [inline src/main/java/bookstore/service/impl/AuthorService.java:6]: Replace **public class AuthorService**  with public class AuthorServiceImpl implements AuthorService
this class implements the service class you created

File: src/main/java/bookstore/service/impl/AuthorService.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package bookstore.service.impl;
+
+import bookstore.model.Author;
+import bookstore.repository.AuthorRepository;
+
```
Comment (Mherr162) [inline src/main/java/bookstore/service/impl/AuthorService.java:5]: Add  @Service
this will inject the service layer to this class

**Review by Mherr162 — COMMENTED** [review]
check out these comments

**Comment by d-herrero** [comment]
Updated based on the comments

**Comment by d-herrero** [comment]
Made some changes in the bookstore controller to allow the creation of an author. Added [authors = new ArrayList<>() under this.biography = biography ] like Mike suggested.

**Comment by d-herrero** [comment]
All four features related to Book Details are completed and work according to my tests.

**Review by Mherr162 — APPROVED** [review]
Well done

**Pull request merged**